cmd/deployment/create.go

```diff
@@ -232,3 +266,44 @@ func setDefaultTemplate(region string) string {
 		return "aws-io-optimized-v2"
 	}
 }
+
+type elasticsearchTopologyElement struct {
+	NodeType  string `json:"node_type"`
+	Size      string `json:"size"`
+	ZoneCount int32  `json:"zone_count,omitempty"`
+}
+
+func esTopologyParseGB(topology []string) ([]string, error) {
+	var t = make([]string, 0, len(topology))
+	for _, rawElement := range topology {
+		var element elasticsearchTopologyElement
+		if err := json.Unmarshal([]byte(rawElement), &element); err != nil {
+			return nil, fmt.Errorf("failed unpacking raw topology: %s", err)
+		}
+
+		if element.Size == "" {
+			return nil, errors.New("memory size cannot be empty")
+		}
+
+		elementSizeMB, err := deploymentsize.ParseGb(element.Size)
```

Comment:
```suggestion
		elementSizeGb, err := deploymentsize.ParseGb(element.Size)
```

Reply:
I named it `elementSizeMB` because the `ParseGB()` function takes the size in GB and returns the size in MB :) So the value of `elementSizeMB` will be megabytes. Does that make sense?

Reply:
ah yes, of course it does, my comment is the one that makes no sense 😅

Reply:
😄